packages/patterns/src/constants.js

```diff
@@ -23,4 +23,6 @@ export const PATTERN_SYNC_TYPES = {
 // TODO: This should not be hardcoded. Maybe there should be a config and/or an UI.
 export const PARTIAL_SYNCING_SUPPORTED_BLOCKS = {
 	'core/paragraph': { content: __( 'Content' ) },
+	'core/heading': { content: __( 'Content' ) },
+	'core/button': { text: __( 'Text' ) },
```

Comment:
Should it also support the URL? It is editable in the UI, but doesn't work at the moment, possibly because it's missing here.

Reply:
Same with the various URL connected attributes such as rel, opening in a new tab etc.

Reply:
If I am not mistaken, the URL of the button should work as well. It was included in the first pull request: [link](https://github.com/WordPress/gutenberg/pull/57249/files#diff-be8a354ce9c01e85c0fdf04c75d76880cdc6c81992db7a86a7a0515dc11a9833R98).

Reply:
Great, let's try adding them in a follow-up PR.

Reply:
`url` support was added in [585a658](https://github.com/WordPress/gutenberg/pull/57789/commits/585a658046cd0e699964fb3f70b261a30fcbadc0)!